src/test/java/org/jabref/preferences/PreviewPreferenceTest.java

```diff
@@ -0,0 +1,41 @@
+package org.jabref.preferences;
+
+import org.junit.Test;
+
+import java.util.List;
+
+public class PreviewPreferenceTest {
+
+    private List<String> previewCycle = null;
+    private int previewCyclePosition = 0;
+    private int previewPanelHeight = 0;
+    private boolean previewPanelEnabled = false;
+    private String previewStyle = "";
+    private String previewStyleDefault = "";
+
+    @Test
+    public void givenNothingWhenCreatingPreviewPreferenceThenNothingThrown() {
```

Comment:
What is the benefit of having this test?
Is it good design that the `PreviewPreferences` class accepts null values in the constructor?
If it isn't, we should not have a test that fails as soon as somebody adds parameter validation...

Reply:
I get your point. 
Implementing this test helped me understand jabref / Preview Preference better. And as there was not a basic test yet, I thought it might be useful for others to see a "skeleton" constructor. 

On another level, to which I have not jet gained enought understanding: May I ask, what is the benefit of not having this test?  If it hinders good design, then I would argue to implement good design, instead of not testing not so good design. Unfortunately, I cannot contribute to this level of discussion or programming yet (I am working on it, though).